AlertaDengue/gis/mapfile.py

```diff
@@ -100,16 +107,17 @@ def __init__(self, **kwargs):
             'map': get_template_content('map.map'),
             'layer': '',
             'url': (
-                settings.MAPSERVER_URL + '?' +
-                'map=/maps/%(map_type)s/%(map_class)s.map&' +
-                'SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&' +
-                'STYLES=&CRS=%(crs_url)s&BBOX=%(bbox)s&' +
-                'WIDTH=%(width)s&HEIGHT=%(height)s&FORMAT=%%(format)s&' +
-                'LAYERS=%(layer_name)s'
+                settings.MAPSERVER_URL  # noqa: F821
```

Comment:
Possivelmente deveria ser
```suggestion
                MAPSERVER_URL
```

Reply:
ok